src/electionguard/decrypt_with_shares.py

```diff
@@ -103,8 +130,14 @@ def decrypt_contest_with_decryption_shares(
     :return: a collection of `PlaintextTallyContest` or `None` if there is an error
     """
     plaintext_selections: Dict[SelectionId, PlaintextTallySelection] = {}
+    selection_description_ids = [
+        description.object_id for description in contest_description.ballot_selections
+    ]
 
     for selection in contest.selections:
+        if selection.object_id not in selection_description_ids and remove_placeholders:
```

Comment:
This is the most important line. 

This handles skipping any placeholders or any other selections not in the Manifest.